local-cli/runAndroid/runAndroid.js

```diff
@@ -215,5 +293,9 @@ module.exports = {
     description: '--flavor has been deprecated. Use --variant instead',
   }, {
     command: '--variant [string]',
+  }, {
+    command: '--deviceId [string]',
+    description: 'builds your app and starts it on a specific device/simulator with the ' +
+      'given device id(listed on "adb devices" command). ',
```

Comment:
nit (wording): given device id (listed by running "adb devices" on the command line).

(no space after the last `.`)